CHANGES/699.feature.2.rst

```diff
@@ -0,0 +1 @@
+Added optional library typing-extensions if Python 3.9 is used. -- by :user Vizonex:
```

Comment:
The role syntax is invalid. Also, we still don't have `:user:` so it wouldn't be interpreted unless `sphinx-issues` is integrated. Additionally, I meant that the trailing period is put after the byline if it's a part of the sentence. Hence the suggested "inline code" syntax for now:
```suggestion
Added optional library ``typing-extensions`` under Python 3.9 runtime or older -- by ``@Vizonex``.
```